dev/restart_all.sh

```diff
@@ -1,6 +1,29 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Yes, but as far as I remember, some features in those scripts are `bash` specific. If it is necessary to support the restricted `sh` we will log an issue